src/resources/styles.ts

```diff
@@ -227,7 +227,7 @@ export const haStyleDialog = css`
     --mdc-dialog-max-width: 600px;
     --mdc-dialog-heading-ink-color: var(--primary-text-color);
     --mdc-dialog-content-ink-color: var(--primary-text-color);
-    --mdc-theme-surface: var(--primary-background-color);
+    --mdc-theme-surface: var(--paper-card-background-color);
```

Comment:
No need to put that in here, it is already derived in `styles.ts`

Reply:
I see, so I should fix it  there instead then, it currently points to `--card-background-color`
Which results in:
![image](https://user-images.githubusercontent.com/15093472/81298014-757db480-9074-11ea-8bbe-5dbb30d3ae04.png)

Reply:
it's probably the theme I'm using that is incomplete, I'll remove it.